hasura/metadata/tables.yaml

```diff
@@ -4,18 +4,27 @@
   is_enum: true
 - table:
     schema: public
-    name: EnneagramType
-  is_enum: true
+    name: ColorAspect
```

Comment:
This should be an enum table, no? we use lowercase for naming normal tables and PascalCase for enum tables